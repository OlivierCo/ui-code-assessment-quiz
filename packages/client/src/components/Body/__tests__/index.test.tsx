import React from 'react'
import { shallow } from 'enzyme'
import { Body } from '../index'

describe('Body', () => {
  it('should be defined', () => {
    expect(Body).toBeDefined()
  })

  it('should render correctly', () => {
    const wrapper = shallow(
      <Body>
        <div className="foo">Bar</div>
      </Body>,
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('should render Body with children', () => {
    const wrapper = shallow(
      <Body>
        <div className="foo">Bar</div>
      </Body>,
    )

    expect(wrapper.contains(<div className="foo">Bar</div>)).toBe(true)
  })
})
