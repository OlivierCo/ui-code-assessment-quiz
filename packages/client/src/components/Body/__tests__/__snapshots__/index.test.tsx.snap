// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body should render correctly 1`] = `
<Fragment>
  <ForwardRef(render)
    styles={
      Object {
        "map": undefined,
        "name": "kh9euv",
        "next": undefined,
        "styles": "body{height: 100%;overflow:hidden;background-color:#fff;color:#000;}",
        "toString": [Function],
      }
    }
  />
  <Container>
    <div
      className="foo"
    >
      Bar
    </div>
  </Container>
</Fragment>
`;
